docs/learn/learn-staking.md

```diff
@@ -601,20 +599,22 @@ dilute over time.
 :::
 
 The ideal staking rate on Polkadot also varies with the number of parachains (50% is the current
-estimation of all DOT that should be staked, per parachain slot).
+estimation of all DOT that should be staked when all the parachain slots are taken).
```

Comment:
Needs more clarity. Is it 50% when running 1 parachain? or is it 50% when running parachains at capacity?

Reply:
According to the w3f research article is 50% when running parachains at capacity, 75% with no parachains.

Reply:
In the current implementation, the ideal staking rate is dynamic and adjusts itself as the number of slots on the relaychain increase. Will summarize that in the doc

https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L535